tests/cache/tests.py

```diff
@@ -1366,6 +1366,19 @@ def test_get_does_not_ignore_non_filenotfound_exceptions(self):
             with self.assertRaises(IOError):
                 cache.get('foo')
 
+    def test_empty_cache_file_does_not_throw_exception(self):
+        cache.set('foo', 'bar')
+        self.assertEqual(cache.get('foo'), 'bar')
+        with open(cache._key_to_file('foo'), 'wb') as fh:
```

Comment:
It seems a more minimal test would be the two with statements only -- the other lines are passing even without the fix -- are they needed?

Reply:
Yeah, you're right. Those tests aren't need. I'll remove them.